src/components/tables/PoolsTable/TokenPills/HiddenTokensPills.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="relative flex mr-2 my-1">
+    <div class="pill" :style="{ zIndex: tokens.length }">
+      {{ $t('tokenPills.hiddenTokens', [tokens.length]) }}
+    </div>
+    <div
+      v-if="hasBalance"
+      class="balance-indicator"
+      :style="{ zIndex: tokens.length }"
+    />
+    <div
+      v-for="n in 2"
```

Comment:
This part renders the "stacked cards" - altho its a loop for `2` items, this may actually change in the future and we might want to render the stacking based on how many hidden tokens there are.